### DISCUSSION TITLE
Recommendations for the shader and what could be applied

### DISCUSSION THREAD

**Discussion by Andriusito** [jebbyk/OSBES-minecraft-bedrock-edition-shader#158] [Ideas]
You know, I also have several recommendations for this shader, I would like to let you know and apply them if you want, would you?

**Comment by Demon-Ak** [comment]
I have an idea that you can add in the shader 
Like waving plants ex: wheat ; carrots ; potato ; 
All of those when they are planted
And how about that when you place for example a glowstone and above it a red glass the light should be red for example

**Comment by Demon-Ak** [comment]
I hope you like my ideas

**Comment by Andriusito** [comment]
also walk the clouds style, in addition to that apply a slightly bluer night as it currently looks quite sad
